### PR TITLE
Update Wrangler command docs

### LINKED ISSUE #1059: Documentation: Add r2 and Pages commands to reference docu
### What version of `Wrangler` are you using?

2.0.2

### What operating system are you using?

Mac

### Describe the Bug

We should add the `wrangler pages` and `wrangler r2` reference documentation to https://developers.cloudflare.com/workers/wrangler/commands/

### PR DESCRIPTION
**Pull request by petebacondarwin**
Add command docs for:

- r2 bucket ...
- pages ...

Closes https://github.com/cloudflare/wrangler2/issues/1059

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -14,8 +14,10 @@ Wrangler offers a number of commands to manage your Cloudflare Workers.
 - [`kv:namespace`](#kvnamespace) - Manage Workers KV namespaces.
 - [`kv:key`](#kvkey) - Manage key-value pairs within a Workers KV namespace.
 - [`kv:bulk`](#kvbulk) - Manage multiple key-value pairs within a Workers KV namespace in batches.
+- [`r2 bucket`](#r2-bucket) - Manage Workers R2 buckets.
 - [`secret`](#secret) - Manage the secret variables for a Worker.
 - [`tail`](#tail) - Start a session to livestream logs from a deployed Worker.
+- [`pages`](#pages) - Configure Cloudflare Pages
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
- [`pages`](#pages) - Configure Cloudflare Pages.
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -785,6 +835,144 @@ After starting `wrangler tail`, you will receive a live feed of console and exce
 
 ---
 
+## pages
+
+Configure Cloudflare Pages.
+
+{{<Aside type="warning">}}
+The `wrangler pages` commands are in beta.<br>
+Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
The `wrangler pages` commands are in beta.
Report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose.
```

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -785,6 +835,144 @@ After starting `wrangler tail`, you will receive a live feed of console and exce
...
+{{</Aside>}}
+
+### `dev`
+
+Develop your full-stack Pages application locally.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
Develop your full stack Pages application locally.
```

File: content/workers/wrangler/commands.md [hunk 4]
```diff
@@ -662,6 +664,54 @@ y
 
 ---
 
+## r2 bucket
+
+Interact with buckets in an R2 store.
+
+{{<Aside type="note">}}
+The `r2 bucket` commands allow you to manage application data in the Cloudflare network to be accessed from Workers using [Workers R2](https://developers.cloudflare.com/workers/runtime-apis/r2/).
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]: @jkup

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (petebacondarwin) [reply content/workers/wrangler/commands.md]: Strictly, `wrangler pages` is not a command, it is a container of sub-commands. How about:

```suggestion
The `wrangler pages ...` commands are in beta.<br>
Report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose.
```

File: content/workers/wrangler/commands.md (hunk 4 above)
Comment (jkup) [reply content/workers/wrangler/commands.md]: @deadlypants1973 maybe should `[Workers R2]` be `[The R2 API]`? Or `the [R2 API]`?

**Review by jkup — APPROVED** [review]
Thanks for doing this!

**Comment by petebacondarwin** [comment]
@deadlypants1973 - do you need to take another look before we merge?

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**